README.md

```diff
@@ -1,23 +1,58 @@
-# MetOfficeAMD
+# MetOfficeDataHub
 Python wrapper around MetOffice Atmospheric Model Data REST API
 
 [![codecov](https://codecov.io/gh/openclimatefix/MetOfficeAMD/branch/main/graph/badge.svg?token=64JOBKZNCI)](https://codecov.io/gh/openclimatefix/MetOfficeAMD)
 
-IN DEVELOPMENT
+MetOfficeAMD is a simple wrapper for the API provided by the British Met Office
+<https://metoffice.apiconnect.ibmcloud.com/metoffice/production/>_ known as Weather DataHub.
```

Comment:
```suggestion
MetOfficeAMD is a simple wrapper for [the API provided by the British Met Office](https://metoffice.apiconnect.ibmcloud.com/metoffice/production/) known as Weather DataHub.
```

Reply:
I feel like using `[this is a link](https://...)` makes it more readable than inserting the link itself